<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brillocontraste</class>
 <widget class="QDialog" name="brillocontraste">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>358</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>358</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Brillo/contraste...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/imagenes/icono.png</normaloff>:/imagenes/icono.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-256</number>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
   <property name="pageStep">
    <number>16</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Brillo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-256</number>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Contraste</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Previsualizar</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(128, 128, 128);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Gama</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
   <property name="tickInterval">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>120</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>brillocontraste</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>brillocontraste</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
